packages/blockchain-wallet-v4-frontend/src/data/alerts/actions.js

```diff
@@ -18,8 +18,11 @@ export const displayInfo = (message, data, persist, timeout) =>
 export const displaySuccess = (message, data, persist, timeout) =>
   display('success', message, data, persist, timeout)
 
-export const displayError = (message, data, persist, timeout) =>
-  display('error', message, data, persist, timeout)
+export const displayError = (message, data, persist, coin, timeout) => {
+  let dismissTimer = timeout
+  if (message === 'iprestriction_login_error') dismissTimer = 9500
```

Comment:
This should not be here, displayError takes a parameter `timeout`, pass it through here https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/1814/files#diff-536ed25300aa632bb5a6f4423c8c5bb3R291